config/prod.exs

```diff
@@ -2,3 +2,13 @@ import Config
 
 config :dotcom, :cache, Dotcom.Cache.Multilevel
 config :dotcom, :trip_plan_feedback_cache, Dotcom.Cache.TripPlanFeedback.Cache
+
+unless System.get_env("PORT") do
+  # configured separately so that we can have the health check not require
```

Comment:
question: Do the health checks run as prod?

Reply:
This was a holdover from many moons ago added here to help Backstop testing that we don't do anymore. https://github.com/mbta/dotcom-legacy/commit/0d8b53a42016da6b5cec3506f79c5fcbb170b095 We can probably revert that to how it was before